QGCCommon.pri

```diff
@@ -57,7 +57,6 @@ linux {
             -Wno-parentheses-equality           # android gstreamer header files
         QMAKE_CFLAGS_WARN_ON += \
             -Wno-unused-command-line-argument   # from somewhere in Qt generated build files
-        QMAKE_LINK += -nostdlib++ # Hack fix?: https://forum.qt.io/topic/103713/error-cannot-find-lc-qt-5-12-android
```

Comment:
Why remove this now?

Reply:
Mentioned here: https://github.com/mavlink/qgroundcontrol/pull/9136#issuecomment-715374359 - not necessary for ndk21, if not removed - link errors under 5.15, either works fine with 5.12